app/src/main/java/com/asfoundation/wallet/ui/iab/payments/carrier/verify/CarrierVerifyFragment.kt

```diff
@@ -156,6 +189,9 @@ class CarrierVerifyFragment : DaggerFragment(), CarrierVerifyView {
     title.visibility = View.INVISIBLE
     disclaimer.visibility = View.INVISIBLE
     phone_number_layout.visibility = View.INVISIBLE
+    if(change_phone_number_button.visibility == View.VISIBLE) {
+      change_phone_number_button.visibility = View.INVISIBLE
+    }
```

Comment:
I think we can avoid the if, if we set it to gone instead of invisible or does it make the view "flick"?

Reply:
The view flicks a bit with visibility set to gone, yes. So it's best to leave it like this